charts/eks-anywhere-packages/values.yaml

```diff
@@ -72,7 +72,9 @@ controller:
   # -- Additional environment variables for the controller pod.
   # - name: EKSA_PUBLIC_KEY
   #   value: ""
-  env: []
```

Comment:
Any particular reason this is an env here rather than a regular value that is an env on the deployment?

Reply:
This does not hide the implementation and will not work if we change the implementation